src/dvc_data/cli.py

```diff
@@ -418,14 +418,15 @@ def merge_tree(oid1: str, oid2: str, force: bool = False):
 def process_patch(patch_file, **kwargs):
     patch = []
     if patch_file:
+        patch_file = Path(patch_file)
```

Comment:
hmm, I expected this to be `pathlib.Path`. Let's fix the definition in click so that we always get a `pathlib.Path`.
Also I noticed that click is returning a relative path instead of absolute path, let's resolve that as well.

https://github.com/iterative/dvc-data/blob/a8a9d3363dd6594ad5c9b7c6bedf1045554495c0/src/dvc_data/cli.py#L483-L488

So what we are missing in above two definitions is that we should set `resolve_path=True` and `path_type=Path`. Let's add that.

```python
cl_path = click.Path(
    exists=True,
    file_okay=True,
    dir_okay=False,
    readable=True,
    path_type=Path,
    resolve_path=True,
)
cl_path_dash = click.Path(
    exists=True,
    file_okay=True,
    dir_okay=False,
    readable=True,
    allow_dash=True,
    path_type=Path,
    resolve_path=True,
)
```

Reply:
Solved in commit [b598151](https://github.com/iterative/dvc-data/pull/127/commits/b5981513eac37338859ca01fe90962f721adbf91)